### DISCUSSION TITLE
Hola Mundo desde Guatemala 🇬🇹

### DISCUSSION THREAD

**Discussion by wuelnerdotexe** [gittogethers/octogatosconf22#53] [Where in the world?]
Un abrazo virtual desde la heeeeerrrmosa capital del Mundo Maya, desde mi hermosa **Guatemala**, el país de la eterna primavera. 🇬🇹

PD: Aquí también habemus DEVs. 🤓

**Comment by isinicolle** [comment]
Holis , saludos desde Honduras hermano guatemalteco

**Reply by wuelnerdotexe** [reply]
¡Hola, qué tal! - saludos hermana y vecina hondureña.
